src/main/java/com/onarandombox/MultiversePortals/commands/ListCommand.java

```diff
@@ -100,12 +89,43 @@ public void runCommand(CommandSender sender, List<String> args) {
 
     private List<String> getPortals(CommandSender sender, MultiverseWorld world, String filter) {
         List<String> portals = new ArrayList<>();
-        if (filter == null)
+        if (filter == null) {
             filter = "";
-        boolean altColor = false;
+        }
         for (MVPortal p : (world == null) ? this.plugin.getPortalManager().getPortals(sender) : this.plugin.getPortalManager().getPortals(sender, world)) {
```

Comment:
I suggest renaming `p` to `portal`, bcu sometime `p` may stand for `player`

Reply:
lol, that already was in the code befor I was editing.